src/create/document.ts

```diff
@@ -45,8 +53,35 @@ export interface ZodOpenApiResponsesObject
     | oas31.ReferenceObject;
 }
 
+export type AnyObjectZodType = ZodType<object, any, any>;
+
+export function getZodObject(schema: AnyObjectZodType): AnyZodObject {
+  if (schema instanceof ZodObject) {
```

Comment:
https://github.com/samchungy/zod-openapi/blob/19df9a67a7df752639161e6ec8ce30509b43e20e/src/create/schema/parsers/index.ts#L64

I had a consumer have issues due to conflicting Zod version's existing in his project. Can we convert these instanceof checks please 🙏

Reply:
oh yeah makes sense, will do